scheduler/src/cook/mesos/api.clj

```diff
@@ -39,6 +39,8 @@
             [cook.mesos.unscheduled :as unscheduled]
             [cook.mesos.util :as util]
             [cook.mesos]
+            [cook.rate-limit :refer [job-submission-rate-limiter]]
```

Comment:
Why the `:refer`? For most things, we tend to favor using a namespace alias and then referencing via the alias instead of with `:refer`

Reply:
This was changed.